engines/dfc_provider/app/controllers/dfc_provider/enterprise_groups_controller.rb

```diff
@@ -3,6 +3,16 @@
 # OFN EnterpriseGroup is represented as DFC Enterprise.
 module DfcProvider
   class EnterpriseGroupsController < DfcProvider::ApplicationController
+    def index
```

Comment:
Hmm I guess this warrants having a separate `enterprise_groups` endpoint (although perhaps it could be achieved with filtering by a "group" tag on enterprises..)

Reply:
There's no standard for filtering in the DFC. Garethe's preferred way is having different endpoints as we do here. But we'll probably add ransack to some endpoints which is outside the standard but not in conflict with it.